veecle-os-runtime-macros/src/lib.rs

```diff
@@ -5,8 +5,59 @@
 mod actor;
 mod storable;
 
-#[allow(missing_docs)]
-// Documentation is defined on the reexport in `../../src/actor.rs` to allow using intra-doc-links.
+/// Generates an [`Actor`] from a function.
+///
+/// [`Actor`]: https://docs.rs/veecle-os/latest/veecle_os/runtime/trait.Actor.html
```

Comment:
```suggestion
/// [`Actor`]: trait.Actor.html
```

works. It's also ugly, but mostly everything ugly in this variant is shared by linking to docs.rs.

Reply:
Yes, but we do not make public docs for that crate so:

* This breaks something hidden to users by default
* But it doesn't make links in public documentation "confusing" (take you to a different site, and potentially a different version, maybe a 404, etc.).

In my opinion, the benefits outweigh the problems.

In any case, I just wanted to point this out; my intention was to approve the PR anyway, but because there was automerge I wanted to be a bit more explicit.

If you think it's best to go with links to docs.rs, resolve this and I'll approve.

Reply:
We'd break [docs.rs for `veecle-os-runtime-macros`](https://docs.rs/veecle-os-runtime-macros/latest/veecle_os_runtime_macros/attr.actor.html) with it (which is currently empty and part of the reason for this PR). I think having a link that works everywhere is a lot better than a link that only works in some specific cases, even though the version might not be consistent.

Reply:
Ah, that's true, but I think if someone is looking at that, they are in the wrong place. However, I suspect there's no way to hide those docs or redirect them to the façade?

Then my preferences are no links > relative link > absolute link to docs.rs, but in any case, I already approved. You can resolve.